### PR TITLE
Upgrade jQuery to ^3.6

### LINKED ISSUE #7806: Main menu scrolling is broken in 4.11.1
### Environment
macOS Monterey, Firefox, Chrome

#### Sonata packages

<details><summary>show</summary>

```
sonata-project/admin-bundle              4.11.1 4.11.1 The missing Symfony Admin Generator
sonata-project/block-bundle              4.12.0 4.12.0 Symfony SonataBlockBundle
sonata-project/cache                     2.2.0  2.2.0  Cache library
Package sonata-project/cache is abandoned, you should avoid using it. No replacement was suggested.
sonata-project/doctrine-extensions       1.16.0 1.16.0 Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 4.2.8  4.2.8  Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/entity-audit-bundle       1.7.0  1.7.0  Audit for Doctrine Entities
sonata-project/exporter                  2.11.0 2.11.0 Lightweight Exporter library
sonata-project/form-extensions           1.13.1 1.13.1 Symfony form extensions
sonata-project/google-authenticator      2.3.1  2.3.1  Library to integrate Google Authenticator into a PHP project
Package sonata-project/google-authenticator is abandoned, you should avoid using it. No replacement was suggested.
sonata-project/intl-bundle               2.12.1 2.12.1 Symfony SonataIntlBundle
sonata-project/twig-extensions           1.9.1  1.9.1  Sonata twig extensions
```
</details>

#### Symfony packages

<details><summary>show</summary>

```
symfony/apache-pack                v1.0.1  v1.0.1  A pack for Apache support in Symfony
symfony/asset                      v6.0.7  v6.0.7  Manages URL generation and versioning of web assets such as CSS stylesheets, JavaScript files and image files
symfony/browser-kit                v6.0.3  v6.0.3  Simulates the behavior of a web browser, allowing you to make requests, click on links and submit forms programmatically
symfony/cache                      v6.0.8  v6.0.8  Provides an extended PSR-6, PSR-16 (and tags) implementation
symfony/cache-contracts            v3.0.1  v3.0.1  Generic abstractions related to caching
symfony/config                     v6.0.8  v6.0.8  Helps you find, load, combine, autofill and validate configuration values of any kind
symfony/console                    v6.0.8  v6.0.8  Eases the creation of beautiful and testable command line interfaces
symfony/css-selector               v6.0.3  v6.0.3  Converts CSS selectors to XPath expressions
symfony/debug-bundle               v6.0.3  v6.0.3  Provides a tight integration of the Symfony VarDumper component and the ServerLogCommand from MonologBridge into the Symfony full-stack framework
symfony/dependency-injection       v6.0.8  v6.0.8  Allows you to standardize and centralize the way objects are constructed in your application
symfony/deprecation-contracts      v2.5.1  v3.0.1  A generic function and convention to trigger deprecation notices
symfony/doctrine-bridge            v6.0.8  v6.0.8  Provides integration for Doctrine with various Symfony components
symfony/dom-crawler                v6.0.6  v6.0.6  Eases DOM navigation for HTML and XML documents
symfony/dotenv                     v6.0.5  v6.0.5  Registers environment variables from a .env file
symfony/error-handler              v6.0.8  v6.0.8  Provides tools to manage errors and ease debugging PHP code
symfony/event-dispatcher           v6.0.3  v6.0.3  Provides tools that allow your application components to communicate with each other by dispatching events and listening to them
symfony/event-dispatcher-contracts v3.0.1  v3.0.1  Generic abstractions related to dispatching event
symfony/expression-language        v6.0.8  v6.0.8  Provides an engine that can compile and evaluate expressions
symfony/filesystem                 v6.0.7  v6.0.7  Provides basic utilities for the filesystem
symfony/finder                     v6.0.8  v6.0.8  Finds files and directories via an intuitive fluent interface
symfony/flex                       v1.18.7 v2.1.8  Composer plugin for Symfony
symfony/form                       v6.0.8  v6.0.8  Allows to easily create, process and reuse HTML forms
symfony/framework-bundle           v6.0.8  v6.0.8  Provides a tight integration between Symfony components and the Symfony full-stack framework
symfony/http-client                v6.0.8  v6.0.8  Provides powerful methods to fetch HTTP resources synchronously or asynchronously
symfony/http-client-contracts      v3.0.1  v3.0.1  Generic abstractions related to HTTP clients
symfony/http-foundation            v6.0.8  v6.0.8  Defines an object-oriented layer for the HTTP specification
symfony/http-kernel                v6.0.8  v6.0.8  Provides a structured process for converting a Request into a Response
symfony/intl                       v6.0.8  v6.0.8  Provides a PHP replacement layer for the C intl extension that includes additional data from the ICU library
symfony/lock                       v6.0.7  v6.0.7  Creates and manages locks, a mechanism to provide exclusive access to a shared resource
symfony/mailer                     v6.0.8  v6.0.8  Helps sending emails
symfony/maker-bundle               v1.41.0 v1.41.0 Symfony Maker helps you create empty commands, controllers, form classes, tests and more so you can forget about writing boilerplate code.
symfony/messenger                  v6.0.8  v6.0.8  Helps applications send and receive messages to/from other applications or via message queues
symfony/mime                       v6.0.8  v6.0.8  Allows manipulating MIME messages
symfony/monolog-bridge             v6.0.3  v6.0.3  Provides integration for Monolog with various Symfony components
symfony/monolog-bundle             v3.7.1  v3.7.1  Symfony MonologBundle
symfony/options-resolver           v6.0.3  v6.0.3  Provides an improved replacement for the array_replace PHP function
symfony/password-hasher            v6.0.8  v6.0.8  Provides password hashing utilities
symfony/phpunit-bridge             v6.0.8  v6.0.8  Provides utilities for PHPUnit, especially user deprecation notices management
symfony/polyfill-intl-grapheme     v1.25.0 v1.25.0 Symfony polyfill for intl's grapheme_* functions
symfony/polyfill-intl-icu          v1.25.0 v1.25.0 Symfony polyfill for intl's ICU-related data and classes
symfony/polyfill-intl-idn          v1.25.0 v1.25.0 Symfony polyfill for intl's idn_to_ascii and idn_to_utf8 functions
symfony/polyfill-intl-normalizer   v1.25.0 v1.25.0 Symfony polyfill for intl's Normalizer class and related functions
symfony/polyfill-mbstring          v1.25.0 v1.25.0 Symfony polyfill for the Mbstring extension
symfony/polyfill-php72             v1.25.0 v1.25.0 Symfony polyfill backporting some PHP 7.2+ features to lower PHP versions
symfony/polyfill-php80             v1.25.0 v1.25.0 Symfony polyfill backporting some PHP 8.0+ features to lower PHP versions
symfony/polyfill-php81             v1.25.0 v1.25.0 Symfony polyfill backporting some PHP 8.1+ features to lower PHP versions
symfony/polyfill-uuid              v1.25.0 v1.25.0 Symfony polyfill for uuid functions
symfony/process                    v6.0.8  v6.0.8  Executes commands in sub-processes
symfony/property-access            v6.0.8  v6.0.8  Provides functions to read and write from/to an object or array using a simple string notation
symfony/property-info              v6.0.7  v6.0.7  Extracts information about PHP class' properties using metadata of popular sources
symfony/proxy-manager-bridge       v6.0.6  v6.0.6  Provides integration for ProxyManager with various Symfony components
symfony/psr-http-message-bridge    v2.1.2  v2.1.2  PSR HTTP message bridge
symfony/rate-limiter               v6.0.8  v6.0.8  Provides a Token Bucket implementation to rate limit input and output in your application
symfony/routing                    v6.0.8  v6.0.8  Maps an HTTP request to a set of configuration variables
symfony/security-acl               v3.3.1  v3.3.1  Symfony Security Component - ACL (Access Control List)
symfony/security-bundle            v6.0.8  v6.0.8  Provides a tight integration of the Security component into the Symfony full-stack framework
symfony/security-core              v6.0.8  v6.0.8  Symfony Security Component - Core Library
symfony/security-csrf              v6.0.3  v6.0.3  Symfony Security Component - CSRF Library
symfony/security-http              v6.0.8  v6.0.8  Symfony Security Component - HTTP Integration
symfony/serializer                 v6.0.8  v6.0.8  Handles serializing and deserializing data structures, including object graphs, into array structures or other formats like XML and JSON.
symfony/serializer-pack            v1.1.0  v1.1.0  A pack for the Symfony serializer
symfony/service-contracts          v3.0.1  v3.0.1  Generic abstractions related to writing services
symfony/stopwatch                  v6.0.5  v6.0.5  Provides a way to profile code
symfony/string                     v6.0.8  v6.0.8  Provides an object-oriented API to strings and deals with bytes, UTF-8 code points and grapheme clusters in a unified way
symfony/templating                 v6.0.3  v6.0.3  Provides all the tools needed to build any kind of template system
symfony/translation                v6.0.8  v6.0.8  Provides tools to internationalize your application
symfony/translation-contracts      v3.0.1  v3.0.1  Generic abstractions related to translation
symfony/twig-bridge                v6.0.8  v6.0.8  Provides integration for Twig with various Symfony components
symfony/twig-bundle                v6.0.8  v6.0.8  Provides a tight integration of Twig into the Symfony full-stack framework
symfony/validator                  v6.0.8  v6.0.8  Provides tools to validate values
symfony/var-dumper                 v6.0.8  v6.0.8  Provides mechanisms for walking through any arbitrary PHP variable
symfony/var-exporter               v6.0.8  v6.0.8  Allows exporting any serializable PHP data structure to plain PHP code
symfony/web-link                   v6.0.3  v6.0.3  Manages links between resources
symfony/web-profiler-bundle        v6.0.8  v6.0.8  Provides a development tool that gives detailed information about the execution of any request
symfony/webpack-encore-bundle      v1.14.1 v1.14.1 Integration with your Symfony app & Webpack Encore!
symfony/yaml                       v6.0.3  v6.0.3  Loads and dumps YAML files
```
</details>

#### PHP version

```
PHP 8.1.5 (cli) (built: Apr 16 2022 00:14:18) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.5, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.5, Copyright (c), by Zend Technologies
    with Xdebug v3.1.2, Copyright (c) 2002-2021, by Derick Rethans
```

## Subject

I have a long main menu with scrolling, after updating from 4.11.0 to 4.11.1 scrolling is not working anymore

## Minimal repository with the bug
4.11.1

## Steps to reproduce
Create many admin pages to reach scrolling in main menu

## Expected results
Main menu can be scrolled to the end

## Actual results
Scrolling isn't working

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this is BC (kind of).

Closes #7806 .
Part of #7156 

## Changelog

```markdown
### Changed
- Update jQuery to `^3.6`
```

### CODE REVIEW

File: src/Resources/public/app.js [hunk 1]
```diff
@@ -1 +1 @@
...r[0]+"With"]=s.fireWith})),i.promise(o),t&&t.call(o,o),o},when:function(t){var e,n,i,o=0,r=a.call(arguments),s=r.length,l=1!==s||t&&g.isFunction(t.promise)?s:0,c=1===l?t:g.Deferred(),u=function(t,n,i){return function(o){n[t]=this,i[t]=arguments.length>1?a.call(arguments):o,i===e?c.notifyWith(n,i):--l||c.resolveWith(n,i)}};if(s>1)for(e=new Array(s),n=new Array(s),i=new Array(s);o<s;o++)r[o]&&g.isFunction(r[o].promise)?r[o].promise().progress(u(o,n,e)).done(u(o,i,r)).fail(c.reject):--l;return l||c.resolveWith(i,r),c.promise()}}),g.fn.ready=function(t){return g.ready.promise().done(t),this},g.extend({isReady:!1,readyWait:1,holdReady:function(t){t?g.readyWait++:g.ready(!0)},ready:function(t){(!0===t?--g.readyWait:g.isReady)||(g.isReady=!0,!0!==t&&--g.readyWait>0||(j.resolveWith(s,[g]),g.fn.triggerHandler&&(g(s).triggerHandler("ready"),g(s).off("ready"))))}}),g.ready.promise=function(t){return j||(j=g.Deferred(),"complete"===s.readyState||"loading"!==s.readyState&&!s.documentElement.doScroll?i.setTimeout(g.ready):(s.addEventListener("DOMContentLoaded",L),i.addEventListener("load",L))),j.promise(t)},g.ready.promise();var M=function(t,e,n,i,o,r,s){var a=0,l=t.length,c=null==n;if("object"=
```
Comment (OskarStark) [inline src/Resources/public/app.js:1]: ?

Comment (jordisala1991) [reply src/Resources/public/app.js:1]: Changing the jQuery version modifies the compiled file, what do you see strange here?

Comment (OskarStark) [reply src/Resources/public/app.js:1]: The diff looks weird, thought it was a mistake, all good then

**Review by VincentLanglet — APPROVED** [review]
Works fine for me

**Pull request merged**